src/hmem_cuda_gdrcopy.c

```diff
@@ -410,4 +449,18 @@ int cuda_gdrcopy_dev_unregister(uint64_t handle)
 	return FI_SUCCESS;
 }
 
+ssize_t ofi_gdrcopy_to_cuda_iov(uint64_t handle, const struct iovec *iov,
+                                size_t iov_count, uint64_t iov_offset,
+								const void *hostptr, size_t len)
```

Comment:
Formatting is off, also name has random changed from const void *host to const void *hostptr, can we keep it consistent?

Reply:
Thanks. Addressed.